src/index.js

```diff
@@ -0,0 +1,38 @@
+import './style.css';
+
+const todoList = [{ description: 'taking breakfast', completed: false, index: '1' },
+  { description: 'taking breakfast 3', completed: false, index: '3' },
+  { description: 'taking breakfast 2', completed: false, index: '2' },
+];
+
+const todos = document.getElementsByClassName('todolist')[0];
+todoList.forEach((todo) => {
+  const task = document.createElement('li');
+  task.classList.add('task');
+  task.id = todo.index;
+  todos.appendChild(task);
+  task.innerHTML = todo.description;
+});
+
+const sortList = () => {
+  let i; let switching; let b; let
+    shouldSwitch;
+  switching = true;
+  while (switching) {
+    switching = false;
+    b = todos.getElementsByTagName('li');
+    for (i = 0; i < (b.length - 1); i += 1) {
+      shouldSwitch = false;
+      if (b[i].innerHTML.toLowerCase() > b[i + 1].innerHTML.toLowerCase()) {
+        shouldSwitch = true;
+        break;
+      }
+    }
+    if (shouldSwitch) {
+      b[i].parentNode.insertBefore(b[i + 1], b[i]);
+      switching = true;
+    }
+  }
+};
```

Comment:
- [ ] You did a good job working on this milestone, however, the logic you choose to sort your list is a bit too complex. Wouldn't it be easier to sort the elements inside the array before displaying them? Instead of sorting them from the already displayed DOM.

If you look closely, your display function iterates over your `todoList` array. What if you create another variable called e.g. `sortedTodoList` and you use the `Array.sort()` function that javascript provides, like this:

```js
const sortedTodoList = todoList.sort((a, b) => a.index - b.index);
```

and then, you iterate over this array instead of the original one to display your todo's, like this:

```js
sortedTodoList.forEach((todo) => {
  const task = document.createElement('li');
  task.classList.add('task');
  task.id = todo.index;
  todos.appendChild(task);
  task.innerHTML = todo.description;
});
```

With this change, your `sortList` function can safely be removed, keeping your code clean and easy to follow!


Final code:

 ```suggestion
const todoList = [{ description: 'taking breakfast', completed: false, index: '1' },
  { description: 'taking breakfast 3', completed: false, index: '3' },
  { description: 'taking breakfast 2', completed: false, index: '2' },
];

const sortedTodoList = todoList.sort((a, b) => a.index - b.index);

// Note this change, document.querySelector returns the first element with the given
// CSS selector, so there's no need for array notation [0] in this case.
const todos = document.querySelector('.todolist');

sortedTodoList.forEach((todo) => {
  const task = document.createElement('li');
  task.classList.add('task');
  task.id = todo.index;
  todos.appendChild(task);
  task.innerHTML = todo.description;
});
```